README.md

```diff
@@ -1,2 +1,76 @@
 # website
 kubesphere website
+
+## Contribute
+
+### Fork and clone the repo
+
+First, create your own fork of the repository.
+
+Then, clone your fork and cd into it:
+
+```
+git clone https://github.com/<username>/website
+cd website
+```
+
+### Running the website locally
+
+hugo version: hugo_extended_0.70.0 + [hugo releases](https://help.github.com/en/desktop/contributing-to-projects/creating-an-issue-or-pull-request#creating-a-new-pull-request)
```

Comment:
why put this link [hugo releases](https://help.github.com/en/desktop/contributing-to-projects/creating-an-issue-or-pull-request#creating-a-new-pull-request) here?

This sentence should be put in a **Prerequisite** section:

### Prerequisite

hugo version: hugo_extended_0.70.0 + , you can install Hugo from xxx.